exetera/core/dataframe.py

```diff
@@ -530,8 +530,31 @@ def to_csv(self, filepath:str, row_filter:Union[np.ndarray, fld.Field]=None, col
                 else:
                     start_row += chunk_row_size
 
-            
-    def drop_duplicates(self, by: Union[str, List[str]], 
+    def to_pandas(self, row_filter=None, col_filter: Union[str, List[str]] = None):
+        """
+        Convert an ExeTera dataframe to Pandas DataFrame.
+        :param row_filter:
+        :param col_filter:
+        :returns: A pandas dataframe.
+
+        Example::
+
+            pandas_df = df.to_pandas()
+        """
+        col_to_convert = list(self._columns.keys()) if col_filter is None else col_filter
+        if isinstance(col_to_convert, list):  # checking data length if multiple columns
+            bench_length = len(self._columns[col_to_convert[0]].data)
+            for field in col_to_convert:
```

Comment:
Could we instead take the minimum length of the columns and convert only that many? Or is having different length columns an error anyway?

Reply:
Yep,  pandas will raise an error for having different length.

Reply:
Line 549 is the only new line here missed by the unit tests according to the coverage report so could you provide a simple test for this failure case?